Framework/src/MonitorObject.cxx

```diff
@@ -111,4 +113,14 @@ void MonitorObject::setDescription(const string& description)
   MonitorObject::mDescription = description;
 }
 
+int MonitorObject::getRunNumber() const
+{
+  return mRunNumber;
+}
+
+void MonitorObject::setRunNumber(int runNumber)
+{
+  MonitorObject::mRunNumber = runNumber;
```

Comment:
Why explicitly?

Reply:
because CLion decided so ;) it is auto-generated. I will fix it.